src/app/component/ProductTabs/ProductTabs.component.js

```diff
@@ -0,0 +1,99 @@
+import PropTypes from 'prop-types';
+import { PureComponent } from 'react';
+
+import ContentWrapper from 'Component/ContentWrapper';
+import { isMobile } from 'Util/Mobile';
+
+import { DEFAULT_TAB } from './ProductTabs.config';
+
+import './ProductTabs.style';
+
+/**
+ * Product tabs
+ * @class ProductTabs
+ * @namespace Component/ProductTabs/Component
+ */
+export class ProductTabs extends PureComponent {
+    static propTypes = {
+        children: PropTypes.node
+    };
+
+    static defaultProps = {
+        children: null
+    };
+
+    state = {
+        activeTab: DEFAULT_TAB
+    };
+
+    onTabClick(e) {
+        const activeTab = e.target.dataset.name;
+
+        this.setState({
+            activeTab
+        });
+    }
+
+    renderActiveTab(activeTab, childrenArray) {
+        return childrenArray.map((item) => {
+            if (item.props.tabName.toLowerCase() === activeTab.toLowerCase()) {
+                return item;
+            }
+
+            return false;
+        });
+    }
+
+    renderAllTabs(childrenArray) {
+        return childrenArray.map((item) => item);
+    }
+
+    renderTabs() {
+        const { children } = this.props;
+        const { activeTab } = this.state;
+
+        const childrenArray = React.Children.toArray(children);
+
+        return (
+            <>
+                <ul
+                  block="ProductTabs"
+                >
+                { childrenArray.map((item) => (
+                    <li
+                      key={ item.props.tabName }
+                      block="ProductTab"
+                      elem="Item"
+                      mods={ { isActive: item.props.tabName.toLowerCase() === activeTab.toLowerCase() } }
+                    >
+                        <button
+                          mix={ { block: 'ProductTab', elem: 'Button' } }
+                          // eslint-disable-next-line react/jsx-no-bind
```

Comment:
Please do not disable the eslint rules.

In this case, to resolve this issue you should declare something like `handleButtonClick` on the class, and use it from the class. 

This rule is introduced because if one declares an arrow function inside of a JSX property, it is getting redeclared every time the component is re-rendered. When you declare it on the class - it does not. 

With big functions this re-declaration during the renders can become memory-intensive pretty quickly. In this specific case it will not, but let's keep the code unitary.

Alternatively, instead of `(e) => this.onTabClick(e)` you can just use `this.onTabClick`.